src/Tizen.NUI/src/public/WindowSystem/QuickPanelService.cs

```diff
@@ -0,0 +1,174 @@
+            if (_tzshQpService == (IntPtr)0)
+            {
+                int err = Tizen.Internals.Errors.ErrorFacts.GetLastResult();
+                _tzsh.ErrorCodeThrow(err);
+            }
+        }
+
+        /// <summary>
+        /// Destructor.
+        /// </summary>
+        /// <exception cref="ArgumentException">Thrown when failed of invalid argument.</exception>
+        /// <since_tizen> 8 </since_tizen>
+        ~QuickPanelService()
+        {
+            Dispose(false);
+        }
+
+        /// <summary>
+        /// Dispose.
+        /// </summary>
+        /// <exception cref="ArgumentException">Thrown when failed of invalid argument.</exception>
+        /// <since_tizen> 8 </since_tizen>
+        public void Dispose()
+        {
+            Dispose(true);
+            GC.SuppressFinalize(this);
+        }
+
+        /// <inheritdoc/>
+        protected virtual void Dispose(bool disposing)
+        {
+            int _res;
+            if (!disposed)
+            {
+                if (_tzshQpService != (IntPtr)0)
+                {
+                    _res = Interop.QuickPanelService.tzshQuickPanelServiceDestroy(_tzshQpService);
+                    _tzsh.ErrorCodeThrow(_res);
+                    _tzshQpService = (IntPtr)0;
+                }
+                disposed = true;
+            }
+        }
+
+        /// <summary>
+        /// Gets the type of the quickpanel service handle.
+        /// </summary>
+        /// <returns>The type of the quickpanel service handle</returns>
+        /// <exception cref="ArgumentException">Thrown when failed of invalid argument.</exception>
+        /// <since_tizen> 8 </since_tizen>
+        public new QuickPanelType GetType()
+        {
+            int _res;
+            IntPtr _type = new IntPtr();
+
+            _res = Interop.QuickPanelService.tzshQuickPanelServiceTypeGet(_tzshQpService, ref _type);
+
+            _tzsh.ErrorCodeThrow(_res);
+
+            return (QuickPanelType)_type;
+        }
+
+        /// <summary>
+        /// Requests to show the quickpanel service window.
+        /// </summary>
+        /// <exception cref="ArgumentException">Thrown when failed of invalid argument.</exception>
+        /// <since_tizen> 8 </since_tizen>
+        public void Show()
+        {
+            int _res;
+
+            _res = Interop.QuickPanelService.tzshQuickPanelServiceShow(_tzshQpService);
+            _tzsh.ErrorCodeThrow(_res);
+        }
+
+        /// <summary>
+        /// Requests to hide the quickpanel service window.
+        /// </summary>
+        /// <exception cref="ArgumentException">Thrown when failed of invalid argument.</exception>
+        /// <since_tizen> 8 </since_tizen>
+        public void Hide()
+        {
+            int _res;
+
+            _res = Interop.QuickPanelService.tzshQuickPanelServiceHide(_tzshQpService);
+            _tzsh.ErrorCodeThrow(_res);
+        }
+    }
+}
```

Comment:
Posix standard define the end of a text line must end with newline character.
if you do not follow this, many text-based tools can make minor problems, so even if there is no problem with compilation, it is better to write newline.

Reply:
I'll fix it